Update heading position on why-fleet.md (#14884)
fleetdm/fleet: docs/Get started/why-fleet.md

- Google Cloud (GCP)
- Azure (Microsoft cloud)
- Data centers

> - Containers (Kube, etc)

Author: ```suggestion
- Containers (kube, etc)
```

- Linux-based IoT devices

## Lighter than air

> Fleet is lightweight and modular.  You can use it for security without MDM and vice versa.  You can turn off features you are not using.

Reviewer: disagree with this line change.  MDM is a capability, and this change makes it less clear

Author: Ok, that makes sense.

> Fleet is lightweight and modular.  You can use it for security without MDM and vice versa.  You can turn off features you are not using.

Author: ```suggestion
Fleet is lightweight and modular.  You can use it for security without using it for MDM, and vice versa.  You can turn off features you are not using.
```

## Lighter than air
Fleet is lightweight and modular.  You can use it for security without using it for MDM, and vice versa.  You can turn off features you are not using.

> ### Openness

Author: ```suggestion
### Open by design
```

Fleet is dedicated to flexibility, accessibility, and clarity.  We think [everyone can contribute](https://fleetdm.com/handbook/company#openness) and that tools should be as easy as possible for everyone to understand.

### Good neighbors

> Fleet has no ambition to replace all of your other tools.  (Though it might replace some if you want it to.)  Ready-to-use, enterprise-friendly [integrations](https://fleetdm.com/integrations) exist for Snowflake, Splunk, GitHub Actions, Vanta, Elastic Jira, Zendesk, and more.

Author: ```suggestion
Ready-to-use, enterprise-friendly [integrations](https://fleetdm.com/integrations) exist for Snowflake, Splunk, GitHub Actions, Vanta, Elastic Jira, Zendesk, and more.
```

Fleet plays well with Munki, Chef, Puppet, and Ansible, as well as with security tools like Crowdstrike and SentinelOne.  For example, you can use the free version of Fleet to quickly report on what hosts are _actually_ running your EDR agent.

> While most folks prefer to use one or the other, Fleet can coexist peacefully with Rapid7 and other agent-based vulnerability scanners.  This can be useful during migrations.

Author: ```suggestion
```

### Free as in free
The free version of Fleet will [always be free](https://fleetdm.com/pricing).  Fleet is [independently backed](https://linkedin.com/company/fleetdm) and actively maintained with the help of many amazing [contributors](https://github.com/fleetdm/fleet/graphs/contributors).

### Longevity

> The [company behind Fleet](https://fleetdm.com/handbook/company) is founded (and majority-owned) by [true believers in open source](https://fleetdm.com/handbook/company/why-this-way#why-open-source).  The company's business model is influenced by GitLab (NYSE: GTLB), with great investors, happy customers, and the capacity to become profitable anytime.

Author: ```suggestion
The [company behind Fleet](https://fleetdm.com/handbook/company) is founded (and majority-owned) by [true believers in open source](https://fleetdm.com/handbook/company/why-this-way#why-open-source).  The company's business model is influenced by GitLab (NYSE: GTLB), with great investors, happy customers, and the capacity to become profitable at any time.
```

In keeping with Fleet's value of openness, [Fleet Device Management's company handbook](https://fleetdm.com/handbook/company) is public and open source.  You can read about the [history of Fleet and osquery](https://fleetdm.com/handbook/company#history) and our commitment to improving the product.

<!-- > To upgrade from Fleet ≤3.2.0, follow the upgrading steps for the earliest subsequent major release from this repository (it'll work out of the box until the release of Fleet 5.0). -->

## Is it any good?
Fleet is used in production by IT and security teams with thousands of laptops and servers.  Many deployments support tens of thousands of hosts, and a few large organizations manage deployments as large as 400,000+ hosts.

## Chat
Please join us in [MacAdmins Slack](https://www.macadmins.org/) or [osquery Slack](https://fleetdm.com/slack).

The Fleet community is full of [kind and helpful people](https://fleetdm.com/handbook/company#empathy).  Whether or not you are a paying customer, if you need help, just ask.

> ## Contributing 

Author: ```suggestion
```

> The landscape of cybersecurity and IT needs to be simplified.  Let's open it up.

Author: ```suggestion
```

> Contributions are welcome, whether you answer questions on [Slack](#chat) / [GitHub](https://github.com/fleetdm/fleet/issues) / [StackOverflow](https://stackoverflow.com/search?q=osquery) / [LinkedIn](https://linkedin.com/company/fleetdm) / [Twitter](https://twitter.com/fleetctl), improve the documentation or [website](https://github.com/fleetdm/fleet/tree/main/website), write a tutorial, give a talk at a conference or local meetup, give an [interview on a podcast](https://fleetdm.com/podcasts), troubleshoot reported issues, or [submit a patch](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Committing-Changes.md).  The Fleet code of conduct is [on GitHub](https://github.com/fleetdm/fleet/blob/main/CODE_OF_CONDUCT.md).

Author: ```suggestion
```

> - Small, iterative, simple (boring) changes are the easiest to merge. -->

Author: ```suggestion
```

## What's next?

### Try it out

[Try out Fleet](https://fleetdm.com/try-fleet) to see what it can do, grab time with one of the maintainers to discuss, or explore the rest of the docs and roll it out to your organization.

> ### Production deployment

Author: ```suggestion
### Questions?
```

> Fleet is simple enough to [spin up for yourself](https://fleetdm.com/docs/deploy/deploy-on-aws-with-terraform).  Or you can have us [host it for you](https://fleetdm.com/pricing).  Premium features are [available](https://fleetdm.com/pricing) either way.

Author: ```suggestion
We have answers to the most [common questions](https://fleetdm.com/docs/get-started/faq).
```
